### DISCUSSION TITLE
problema codice sensore, ne fa sparire un altro.

### DISCUSSION THREAD

**Discussion by motogm** [indomus/forum#1235] [GENERALE - Discussioni generiche]
ciao a tutti. ho preso un ecovacs cosi ho scaricato l'app e collegato ad Home assistant. Ora volevo seguire la procedura per farlo attivare quando non siano a casa passate tot ore dall'ultima pulizia, seguendo la guida presente su indomus a questo lin
https://indomus.it/progetti/pulire-automaticamente-casa-quando-si-esce-tramite-la-domotica-di-home-assistant/
nel configuration.aml ho quindi inserito il codice per creare la durata del ciclo di aspirazione che riporto qui:

`#Sensore aspirapolvere in pausa
sensor:
  - platform: history_stats
    name: Durata ciclo aspirapolvere
    entity_id: vacuum.rosita
    state: 'docked'
    type: ratio
    end: '{{ now() }}'
    duration:
      hours: 12`
Ho fatto convalida ed è andato tutto ok. Il problema è che quando ho riavviato HA, mi è sparito il sensore virtuale di consumo che fa la solla della produzione fotovoltaico + richiesta abitazione dandomi la reale disponibilità/consumo.
Il codice del sensore virtuale è:
`# Consumi istantanei
sensor:
  - platform: template
    sensors:
      powermon_total:
        friendly_name: "Potenza istantanea totale"
        icon_template: hass:gauge
        value_template: "{{ (states('sensor.shellyem3_XXXXXXX_channel_a_power') | float + states('sensor.shellyem3_XXXXXX_channel_b_power') | float) }}"
        unit_of_measurement: 'Watt'`
Mi sparisce l'entità sensor.powermon_total, ma se cancello il codice della durata ciclo aspirazione, torna tutto a funzionare. Non riesco a capire che correlazione ci sia tra i due.

`![Schermata 2021-12-27 alle 16 42 45](https://user-images.githubusercontent.com/83405414/147486974-8f3b0329-2e7e-48fb-a87e-297ffdf7a46e.png)`

Potete aiutarmi a capire?

**Comment by snakuzzo** [comment]
Ripetiamo per l'ennesima volta nell'ennesimo topic di adeguare il contenuto del post utilizzando il markdown per il codice.
Qui è scritto come fare -> https://github.com/indomus/forum/blob/main/README.md
Grazie
